spring-cloud-stream/src/main/java/org/springframework/cloud/stream/config/SpringIntegrationProperties.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2017 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.stream.config;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.messaging.MessageHeaders;
+
+/**
+ * Contains properties for Spring Integration settings.
+ */
+@ConfigurationProperties("spring.cloud.stream.integration")
+public class SpringIntegrationProperties {
+
+	private String[] messageHandlerNotPropagatedHeaders = new String[] { MessageHeaders.CONTENT_TYPE };
```

Comment:
This probably should go to `IntegrationProperties` in Spring Boot 2.0: https://github.com/spring-projects/spring-boot/blob/master/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/integration/IntegrationProperties.java

Please, consider to add some JavaDocs for Boot meta-data any way: http://docs.spring.io/spring-boot/docs/1.5.3.RELEASE/reference/html/configuration-metadata.html#configuration-metadata-annotation-processor